images/php/settings.php

```diff
@@ -347,7 +347,14 @@
 }
 
 // Override data_pipelines url.
-$config['data_pipelines.dataset_destination.sdp_elasticsearch']['destinationSettings']['url'] = (getenv('SEARCH_HASH') && getenv('SEARCH_URL')) ? sprintf('http://%s.%s', getenv('SEARCH_HASH'), getenv('SEARCH_URL')) : "http://elasticsearch:9200";
+$elasticsearch_url = (getenv('SEARCH_HASH') && getenv('SEARCH_URL')) ? sprintf('http://%s.%s', getenv('SEARCH_HASH'), getenv('SEARCH_URL')) : "http://elasticsearch:9200";
+$data_pipeline_configs = [
+  'sdp_elasticsearch',
+  'sdp_elasticsearch_v1',
```

Comment:
I meant to ask in the other PR.... what's `v1` for?

Reply:
`v1` represents version 1 :stuck_out_tongue_winking_eye: 
`sdp_elasticsearch_v1` indicates it uses the `Destination` plugin that defined by `tide_data_pipeline` module.

Reply:
We've had to override the existing plugin to get new functionality?

Reply:
The plan of mine is for these 2 types of **Destination**s to coexist for now. If it is a new data pipeline or needs to be switched to v1, we only need to select v1 in the `datapipeline content`

Reply:
I don't have enough context to approve this PR. @anthony-malkoun are you happy with Vincent's justification?